gensim/models/doc2vec.py

```diff
@@ -82,6 +82,8 @@
 
 logger = logging.getLogger(__name__)
 
+SEED_DIVERSIFIER = 7919  # 1000th prime
```

Comment:
Historically, I didn't perceive a strong public/private distinction in Gensim code, though I've seen the leading-`_` convention growing recently. Yes, this is wholly internal to `Doc2Vec` at the moment, and only used in one place, so the literal number could be used that one place instead with a code comment to explain its role (instead of its variable name).

Reply:
I went ahead an inlined the seed-offset, as it's only done in one place.